README.md

```diff
@@ -172,6 +172,14 @@ export IMMUDB_S3_ENDPOINT="https://${IMMUDB_S3_BUCKET_NAME}.s3.${IMMUDB_S3_LOCAT
 ./immudb
 ```
 
+When working with the external storage, you can enable the option for the remote storage
+to be the primary source of identifier. This way, if ImmuDB is run using ephemeral disks,
```

Comment:
ultra minor, we prefer just lower-case name for immudb e.g. `This way, if immudb is run ...`